app/code/Magento/Deploy/Model/Filesystem.php

```diff
@@ -277,12 +277,18 @@ protected function compile(OutputInterface $output)
      * Deletes specified directories by code
      *
      * @param array $directoryCodeList
+     * @param bool $resetStaticVersion
      * @return void
      * @throws \Magento\Framework\Exception\FileSystemException
      */
-    public function cleanupFilesystem($directoryCodeList)
+    public function cleanupFilesystem($directoryCodeList, $resetStaticVersion = false)
```

Comment:
Unfortunately, we are not allowed to change public method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)